tests/rpc-tests/rpc-compat.js

```diff
@@ -0,0 +1,199 @@
+import fs from 'fs'
+import path from 'path'
+import { maxRetries, runScpCommand, runSshCommand, runSshCommandWithReturn } from '../../src/express/common/remote-worker'
+import assert from 'assert'
+import { checkAndReturnVMIndex, loadDevnetConfig } from '../../src/express/common/config-utils'
+import HDWalletProvider from '@truffle/hdwallet-provider'
+import { fundAccount } from '../test-eip-1559'
+import { timer } from '../../src/express/common/time-utils'
+const axios = require('axios/dist/node/axios.cjs')
+const Web3 = require('web3')
+
+require('dotenv').config({ path: `${process.cwd()}/.env` })
+
+async function initWeb3(machine) {
+  const provider = new HDWalletProvider({
+    mnemonic: {
+      phrase: process.env.MNEMONIC
+    },
+    providerOrUrl: `http://${machine}:8545`
+  })
+
+    return new Web3(provider)
+}
+
+async function init() {
+  console.log('📍Executing RPC compatibility tests')
+  const doc = await loadDevnetConfig('remote')
+  let machine = doc.devnetBorHosts[0]
```

Comment:
can we run these tests on all the nodes of the devnet?

Reply:
Well, practically speaking, we'll only be spinning up 1 node since it's going to run in archive mode. And I guess running 1 node should suffice.